### ISSUE TITLE
Accessibility: MAS05A: Keyboard: Keyboard focus is not visible for the "Skip to Content" link .

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#40]
#A11yMAS
#MAS05A
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in Edge.
2. Navigate through the page.
3. Observe the  keyboard focus for the "Skip to Content" link.

**Actual Result:**
Keyboard focus is not visible for the "Skip to Content" link.
**_Narrator Behavior:_**
The Narrator announces "Skip to Content" link though the link is not visible.

**Expected Result:**
Keyboard focus should be visible for the "Skip to Content" link.

**User Impact:**
Users with visual impairments will be misguided with the wrong announcement of the screen reader

**Screenshots:**
![keyboard focus is not visible for the skip to content link](https://cloud.githubusercontent.com/assets/24825441/21713181/96fb1a14-d41e-11e6-9d7f-803732a42d1a.png)

**Comment by waderyan** [comment]
I cannot reproduce. 
![image](https://cloud.githubusercontent.com/assets/2449568/26378485/8f4ec4f2-3fc9-11e7-9bdf-256a419e4fe1.png)
